smelter-core/src/pipeline/v4l2/mod.rs

```diff
@@ -0,0 +1,270 @@
+use std::sync::{Arc, atomic::AtomicBool};
```

Comment:
try to keep mod file small

There are few other places that we have like that, but they are usually effect of moving existing files.